src/TroykaOLED.cpp

```diff
@@ -10,14 +10,18 @@
 
 #include "TroykaOLED.h"
 
+#define _AUTO_UPDATE()      \
+    if (_stateAutoUpdate) { \
+        update();           \
+    }
+
 TroykaOLED::TroykaOLED(uint8_t i2cAddress, uint8_t width, uint8_t height) {
     _i2cAddress = i2cAddress;
     _width = width;
     _height = height;
-    _bufferDisplay = new uint8_t[_width * _height / 8];
     _stateInvert = false;
-    _stateAutoUpdate = true;
-    _stateImageBG = true;
+    _stateAutoUpdate = false;
```

Comment:
it`s better to use _stateAutoUpdate = true as default, because all our old sketches will work without update()

Reply:
Good Idea. Done in commits history end.